internal/action/otp.go

```diff
@@ -31,7 +31,14 @@ func (s *Action) OTP(c *cli.Context) error {
 	}
 
 	qrf := c.String("qr")
-	clip := c.Bool("clip")
+	clip := config.Bool(ctx, "otp.onlyclip")
+	if c.IsSet("clip") {
+		clip = c.Bool("clip")
+	}
+	alsoClip := config.Bool(ctx, "otp.autoclip")
+	if c.IsSet("alsoclip") {
```

Comment:
You're using the `alsoclip` flag, so you should change the [docs otp.md](docs/commands/otp.md) file to reflect the right information. Right now the docs says `--autoclip` instead of `--alsoclip`